src/plugins/crypto/CMakeLists.txt

```diff
@@ -1,20 +1,52 @@
-find_package (LibGcrypt)
+# set default value to CRYPTO_API
+if (NOT DEFINED CRYPTO_API OR CRYPTO_API STREQUAL "")
+	set (CRYPTO_API "gcrypt")
+	message ("Trying to use gcrypt as default crypto API")
+endif (NOT DEFINED CRYPTO_API OR CRYPTO_API STREQUAL "")
 
-if (LIBGCRYPT_FOUND)
-	configure_file (
-		"${CMAKE_CURRENT_SOURCE_DIR}/crypto_internal.h.in"
-		"${CMAKE_CURRENT_BINARY_DIR}/crypto_internal.h"
-	)
+set (ELEKTRA_CRYPTO_API_GCRYPT FALSE)
+set (ELEKTRA_CRYPTO_API_OPENSSL FALSE)
```

Comment:
With compilation variant it is possible to recompile everything with different macros set.
It might be not worth the effort if you think openssl is superior anyway.